.github/workflows/stale-branches

```diff
@@ -0,0 +1,24 @@
+# .github/workflows/stale-branches.yml
+
+name: Stale Branches
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+    
+permissions:
+  issues: write
+  contents: write
+
+jobs:
+  stale_branches:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Stale Branches
+      uses: crs-k/stale-branches@v5.0.0
+      with:
+        repo-token: '${{ secrets.GITHUB_TOKEN }}'
+        days-before-stale: 180
+        days-before-delete: 360
+        branches-filter-regex: ''^(?!release/miner)'
```

Comment:
is this regex right? we'd want everything starting with `release` right, so `releases`, `release/vx.y.z`, `release/miner/vx.y.z` so just chopping off the `/miner` part of this should be good?

Reply:
My thought was that release branches is already protected by branch protection rules, so with this pattern we could verify that that branch protection pattern also works. But chopping off the `/miner` should be fine, and there is definetly other branches that has protection rules on them, which we can verify that with. Updated here: https://github.com/filecoin-project/lotus/pull/12858/commits/1e66f3e00530c6edd461befa7a5b2c0636fe8319